repos/system_upgrade/common/actors/repositoriesmapping/tests/unit_test_repositoriesmapping.py

```diff
@@ -133,31 +317,14 @@ def __call__(self, *args, **kwargs):
         return ResponseMocked(self.status_code, self.text)
 
 
-@pytest.mark.parametrize('isFile', (False, True))
-def test_scan_missing_or_empty_file(monkeypatch, isFile):
-    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(architecture.ARCH_X86_64))
-    monkeypatch.setattr(api, 'produce', produce_mocked())
-    monkeypatch.setattr('os.path.isfile', lambda dummy: False)
-    monkeypatch.setattr(requests, 'get', get_mocked(200 if isFile else 404, ''))
-
-    with pytest.raises(StopActorExecutionError) as err:
-        repositoriesmapping.scan_repositories()
-    assert not api.produce.called
-    assert 'invalid or could not be retrieved' in str(err)
-
-
-@pytest.mark.parametrize('line', [
-    'whatever invalid line',
-    'something, jaj',
-    'a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v',
-    'valid,',
-    '7-server-rpms,8-server-rpms,name-8-repo-rpms,all,all,x86_64,rpm',
-    '7-server-rpms,8-server-rpms,name-8-repo-rpms,all,all,x86_64,rpm,ga,ga,invalid',
-])
-def test_scan_invalid_file_csv(monkeypatch, line):
-    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(architecture.ARCH_X86_64))
-    monkeypatch.setattr(api, 'produce', produce_mocked())
-    with pytest.raises(StopActorExecutionError) as err:
-        repositoriesmapping.scan_repositories(read_repofile_func=ReadRepoFileMock([line]))
-    assert not api.produce.called
-    assert 'The repository mapping file is invalid' in str(err)
+# # @pytest.mark.parametrize('isFile', (False, True))
+# # def test_scan_missing_or_empty_file(monkeypatch, isFile):
+# #     monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(architecture.ARCH_X86_64))
+# #     monkeypatch.setattr(api, 'produce', produce_mocked())
+# #     monkeypatch.setattr('os.path.isfile', lambda dummy: False)
+# #     monkeypatch.setattr(requests, 'get', get_mocked(200 if isFile else 404, ''))
+# #
+# #     with pytest.raises(StopActorExecutionError) as err:
+# #         repositoriesmapping.scan_repositories()
+# #     assert not api.produce.called
+# #     assert 'invalid or could not be retrieved' in str(err)
```

Comment:
This could be reintroduced, however, it seems more like a test for `read_or_fetch` than the actor itself.

Reply:
partially. the function has a behaviour in such a cases (in case of missing file it's a different function). but we need to test how the actor behave in such a case. Whether it produces expected output or just crashes.